lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -11,84 +10,58 @@ class ModuleDocFragment(object):
     # Standard documentation fragment
     DOCUMENTATION = r'''
 attributes:
-    action:
-      description: Indicates this has a corresponding action plugin so some parts of the options can be executed on the controller
-      support: none
+    check_mode:
+      description: Can run in check_mode and return changed status prediction withought modifying target
+    diff_mode:
+      description: Will return details on what has changed (or possibly needs changing in check_mode), when in diff mode
+    platform:
+      description: Target OS/families that can be operated against
+'''
+
+    ACTIONGROUPS = r'''
+attributes:
     action_group:
       description: Action is part of action_group(s), for convenient setting of module_defaults.
       support: none
       membership: []
-    api:
-      description: Instead of executing code on a target, this action interacts with an API on behalf of the target.
-      support: none
-    async:
-      description: Supports being used with the ``async`` keyword
-      support: full
+'''
+
+    CONN = r'''
+attributes:
     become:
       description: Is usable alongside become keywords
-      support: full
-    bypass_host_loop:
-      description: Forces a 'global' task that does not execute per host, this bypasses per host templating and serial,
-                   throttle and other loop considerations.  Also, this action cannot be used in non lockstep strategies
-      support: none
-    check_mode:
-      description: Can run in check_mode and return changed status prediction withought modifying target
-      support: none
     connection:
       description: Uses the target's configured connection information to execute code on it
-      support: full
-    conditional:
-      description: Will respect the `when` keyword  per item loop or task (when no loop is present)
-      support: full
     delegation:
       description: Can be used in conjunction with delegate_to and related keywords
-      support: full
-    diff_mode:
-      description: Will return details on what has changed (or possibly needs changing in check_mode), when in diff mode
-      support: none
+'''
+
+    FACTS = r'''
+attributes:
     facts:
       description: Action returns an ``ansible_facts`` dictionary that will update existing host facts
-      support: none
-    forced_local:
-      description: The connection itself is passed to the action while the code is still executed on the controller
-      support: none
-    forced_action_plugin:
```

Comment:
If `forced_action_plugin` is completely removed, shouldn't the corresponding part in the schema be removed as well?

Reply:
no, i expect this to be common for networking modules, removed it from the core file since nothing in core uses it.

Reply:
IMO this really should be defined in action_common_attributes, to help ensuring that all collections use the same description there.

Reply:
others have suggested that network-common or utils collections are a better home, since nothing in core currently uses it I've decided to trim away all such options (same with cloud/api/turbo/etc)